features/environment.py

```diff
@@ -128,13 +129,18 @@ def _make_patroni_test_config(self, name, dcs, tags):
             config['tags'] = tags
 
         if dcs != 'etcd':
-            etcd = config.pop('etcd')
-            config['zookeeper'] = {'scope': etcd['scope'], 'session_timeout': etcd['ttl'],
-                                   'reconnect_timeout': config['loop_wait']}
-            if dcs == 'exhibitor':
-                config['zookeeper']['exhibitor'] = {'hosts': ['127.0.0.1'], 'port': 8181}
+            dcs_config = config.pop('etcd')
```

Comment:
Actually, the code in __init__.py from Patroni will take the first one of etcd, zookeeper and consul available, so instead of taking the etcd configuration and changing it to fit consul or zookeeper we should change the sample configuration to have values for all three.